### PR TITLE
[ci skip] Better description for starting up with new Rails project

### PR DESCRIPTION
**Pull request by shivanibhanwal**
These changes rephrases the description with better explanation of step by step process.

### CODE REVIEW

**Comment by jonatack** [comment]
Agreed that it could be better written, but the meaning of the sentence is changed and the main message arguably lost.

**Comment by shivanibhanwal** [comment]
@jonatack as per my understanding earlier message is to mention that we just need to follow along to make the app run and there no extra code or step needed because nothing has been left out everything is included. Isn't that right? Please correct me if I did not understand it correctly.

**Comment by jonatack** [comment]
No problem. The main message for me was that all the steps in the tutorial are important / non-optional (and so they should be followed step by step).

**Comment by robin850** [comment]
The last sentence seems a bit redundant with the previous one. What do you think about:

> The best way to read this guide is to follow it step by step. No additional 
> code or step is needed to run this example application.

**Comment by shivanibhanwal** [comment]
@jonatack , @robin850 thanks for your inputs. I have rephrased it as per your suggestion
Let me know what you think?

**Comment by mtthwhggns** [comment]
The sentence:

> As all steps are important/non-optional.

Could be changed to something like:

> All steps are important and essential.

In this sentence, the 'As' is like 'because', so it doesn't quite make sense when the rest of the old sentence has been split into another sentence. 

I personally don't like the term non-optional, but that's a matter of preference rather than anything linguistic.

**Comment by shivanibhanwal** [comment]
@mtthwhggns Thanks for the good suggestion :+1: and I liked it and made changes

File: guides/source/getting_started.md [hunk 1]
```diff
@@ -71,9 +71,8 @@ The Rails philosophy includes two major guiding principles:
 Creating a New Rails Project
 ----------------------------
 
-The best way to use this guide is to follow each step as it happens, no code or
-step needed to make this example application has been left out, so you can
-literally follow along step by step.
+The best way to read this guide is to follow it step by step. All steps are important
```
Comment (robin850) [inline guides/source/getting_started.md]: Can you wrap your additions at 80 chars and remove the whitespace at the end of the line please ?

File: guides/source/getting_started.md [hunk 2]
```diff
@@ -71,9 +71,8 @@ The Rails philosophy includes two major guiding principles:
 Creating a New Rails Project
 ----------------------------
 
-The best way to use this guide is to follow each step as it happens, no code or
-step needed to make this example application has been left out, so you can
-literally follow along step by step.
+The best way to read this guide is to follow it step by step. All steps are important 
+and essential, to run this example application and no additional code or steps are needed.
```
Comment (robin850) [inline guides/source/getting_started.md]: I guess you can remove the comma here.

File: guides/source/getting_started.md (hunk 1 above)
Comment (shivanibhanwal) [inline guides/source/getting_started.md]: In online editor, I just see two option for wrapping "No Wrap" or "soft wrap". So not sure how specifically I should set it for 80 chars. However I am doing that manually.

**Comment by vijaydev** [comment]
Please squash the commits into one.

**Comment by robin850** [comment]
I took the liberty to manually apply it at https://github.com/rails/rails/commit/ac1c888b95481cb66df8175952403b447fffe7e8, removing the white spaces at the end of the lines ; thanks for your contribution ! :-)

**Comment by shivanibhanwal** [comment]
Thank you @robin850 !!

**Pull request closed without merging**